### PR TITLE
Header component

### PR DESCRIPTION
**Pull request by ceciliahangzhou**
Create Header Component 
[BVPS-136](https://deloittedigital.atlassian.net/browse/BVPS-136)
[BVPS-137](https://deloittedigital.atlassian.net/browse/BVPS-137)

### CODE REVIEW

File: components/Alert/Alert.stories.jsx [hunk 1]
```diff
@@ -1,8 +1,8 @@
-import { Alert } from '.'
+import { Header } from '.'
```
Comment (emilymendelson) [inline components/Alert/Alert.stories.jsx:1]: Should this file be changed?

File: components/Header/index.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import PropTypes from 'prop-types'
+import './styles.css'
+import image from '../../assets/images/logo-banner.svg'
+
+export const Header = ({
```
Comment (emilymendelson) [inline components/Header/index.jsx:5]: I think we would also need a unit test file for the header component

**Review by hannah-macdonald1 — COMMENTED** [review]
Code looks good but I don't see any tests... do we need some here?

File: components/Footer/index.jsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+		</div>
+	)
+}
+
+
+Footer.defaultProps = {
+	links: [
+		{
+			name: 'Home', 
+			href: ''
```
Comment (habibaz) [inline components/Footer/index.jsx:27]: @ceciliahangzhou Can we please make the items Dynamic. Also label/value should read from a separate file.

File: components/Alert/Alert.stories.jsx (hunk 1 above)
Comment (hannah-macdonald1) [reply components/Alert/Alert.stories.jsx:1]: Second this. Should the file be called Header instead of Alert?

**Pull request merged**